source/_integrations/generic.markdown

```diff
@@ -91,85 +77,65 @@ In this section, you find some real-life examples of how to use this camera plat
 
 ### Weather graph from yr.no
 
-```yaml
-camera:
-  - platform: generic
-    name: Weather
-    still_image_url: https://www.yr.no/en/content/1-72837/meteogram.svg
-    content_type: "image/svg+xml"
-```
+| Parameter | Setting |
```

Comment:
Please avoid tables when possible

Reply:
I don't mind either way but tables were suggested by @bdraco.

What is the best alternative to tables?

Reply:
Ok, done.